Resources/config/controller.xml

```diff
@@ -16,14 +16,15 @@
             <tag name="controller.service_arguments" />
         </service>
 
-        <service id="HWI\Bundle\OAuthBundle\Controller\LoginController" public="true">
+        <service id="HWI\Bundle\OAuthBundle\Controller\LoginController" public="true" autowire="true">
```

Comment:
so @niels-nijens I discussed it with @nicolas-grekas and we found a solution:

we can remove `autowire="true"` and add `<call method="setContainer" />` since there is a DI binding for the service locator :blush: I just tried it myself and it works. So the correct service locator is injected then.

Reply:
Ah, nice! Changing back then. 😄